OOVPA.h

```diff
@@ -172,6 +178,27 @@ typedef struct _OOVPATable
  #define REGISTER_OOVPA_18(Symbol, Version, ...) REGISTER_OOVPA(Symbol, Version), MSVC_EXPAND(REGISTER_OOVPA_17(Symbol, __VA_ARGS__))
  #define REGISTER_OOVPA_19(Symbol, Version, ...) REGISTER_OOVPA(Symbol, Version), MSVC_EXPAND(REGISTER_OOVPA_18(Symbol, __VA_ARGS__))
 
+#define OV_ENTRY_0(...)
+ #define OV_ENTRY_1(Offset, Value) OV_ENTRY(Offset, Value)
+ #define OV_ENTRY_2(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_1(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_3(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_2(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_4(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_3(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_5(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_4(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_6(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_5(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_7(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_6(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_8(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_7(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_9(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_8(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_10(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_9(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_11(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_10(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_12(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_11(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_13(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_12(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_14(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_13(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_15(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_14(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_16(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_15(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_17(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_16(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_18(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_17(Offset+1, __VA_ARGS__))
+ #define OV_ENTRY_19(Offset, Value, ...) OV_ENTRY(Offset, Value), MSVC_EXPAND(OV_ENTRY_18(Offset+1, __VA_ARGS__))
```

Comment:
nit: I believe the longest x86 instruction can only be 16 bytes; so 19 bytes might be overkill - not worth changing though.

Reply:
At least it give you ability to add second instruction in same line. Assuming, it's not over 19 bytes.

We had put a comments with 2+ instructions in one comment. Though, not every values are used, some are commented out to reduce the count and is not unique values.